src/gafaelfawr/providers/oidc.py

```diff
@@ -223,6 +280,9 @@ async def _get_ldap_groups(self, uid: str) -> List[TokenGroup]:
         member_attr = self._ldap_config.group_member
         ldap_query = f"(&(objectClass={group_class})({member_attr}={uid}))"
 
+        self._logger.debug(
+            f"querying ldap {self._ldap_config.url} at {self._ldap_config.base_dn} for groups with {ldap_query}"
```

Comment:
Same note here on long lines.

Reply:
fixed in bf08596ea67e12fc518bb883ae340b8c4b52a2bc